webrtc.html

```diff
@@ -2385,82 +2386,95 @@
       <h3>Interfaces for Connectivity Establishment</h3>
 
       <section>
-        <h4>RTCIceCandidate Dictionary</h4>
+        <h4>RTCIceCandidate Interface</h4>
 
-        <p>This describes an ICE candidate. Though not explicitly required,
-        values for <var>candidate</var> and either <var>sdpMid</var>
-        or <var>sdpMLineIndex</var> MUST be provided.</p>
+        <p>This interface describes an ICE candidate.</p>
 
-        <dl class="idl" title="[Constructor(RTCIceCandidate copy)] dictionary RTCIceCandidate">
-          <dt>required DOMString candidate</dt>
+        <dl class="idl" data-merge="RTCIceCandidateInit" title="interface RTCIceCandidate">
+          <dt>Constructor (RTCIceCandidateInit candidateInitDict)</dt>
+
+          <dd>The <dfn id="dom-icecandidate"><code>RTCIceCandidate()</code></dfn> constructor
+          takes a dictionary argument, <var>candidateInitDict</var>, whose content is used
+          to initialize the new
+          <a class="idlType" href="#idl-def-RTCIceCandidate"><code>RTCIceCandidate</code></a>
+          object. If <var>sdpMid</var> or <var>sdpMLineIndex</var> is not present in
+          <var>candidateInitDict</var>, the corresponding attribute will be initialized to
+          <code>null</code>.</dd>
+
```

Comment:
When I read it again, I see that my previous comment wasn't right.

But the constructor needs to throw as well. A straight forward way to define it would be to set each property (sdpMid and sdpMLineIndex) to null if not present in the init dict. And follow up with a step that throws if both are null.